### ISSUE TITLE
Discussion sur la DB

### ISSUE THREAD

**Issue by Samuel-Scalbert** [Samuel-Scalbert/SOFTware-Viz#14]
Étant donné que nous avons des fichiers XML (contenant les mentions de logiciels), il serait judicieux de modifier la base de données et de la nourrir uniquement avec ces fichiers XML. Pour ce faire, il est également nécessaire de modifier SOFTware-Sync afin d'ajouter dans les fichiers XML toutes les données dont nous avons besoin.

**Comment by AlainMonteil** [comment]
Actuellement c'est du Json resultat de Softcite c'est ça ? 
L'dée serai d'avoir une BD xml avec les TEI des pdf en intégrant le résultat se softcite ? 
Comme je ne comprend pas l'erreur #15 c'est à dire pourquoi in a associé SEP2P a hal-03404125 alors qu'il n'y a pas de liens et du coup n'affiche rien en context ... j'espère que je ne me trompe pas et que c'est pour éviter ce type d'erreur

**Comment by Samuel-Scalbert** [comment]
Oui, la base de données est construite avec les fichiers XML d'HAL et les fichiers JSON de softcite. L'objectif est quand même, à terme, d'utiliser plutôt les résultats de SOFTware-Sync, car les doublons ou les mentions de logiciels avec des erreurs sont évités.

Pour SEP2P, tu devrais avoir une page web avec environ dix fois le même contexte.

**Comment by AlainMonteil** [comment]
Ah non je n'ai pas cela j'ai 2 identifiant Hal et les deux page donne le context

**Comment by AlainMonteil** [comment]
sinon ok pour la DB
